common/authorization/interceptor.go

```diff
@@ -212,11 +216,17 @@ func NewAuthorizationInterceptor(
 // getMetricsHandler return metrics handler with namespace tag
 func (a *interceptor) getMetricsHandler(
 	operation string,
-	namespace string,
+	nsName string,
 ) metrics.Handler {
 	var metricsHandler metrics.Handler
-	if namespace != "" {
-		metricsHandler = a.metricsHandler.WithTags(metrics.OperationTag(operation), metrics.NamespaceTag(namespace))
+	if nsName != "" {
+		if entry, _ := a.namespaceRegistry.GetNamespace(namespace.Name(nsName)); entry == nil {
+			// if this isn't a known namespace, do not use it as a tag value, to avoid cardinality issues
+			nsName = ""
```

Comment:
shall we use some const value like "UNKNOWN"?

Reply:
this falls into the path below that uses `metrics.NamespaceUnknownTag()`, which ends up as `_unknown_`